hip_data_tools/etl/athena_to_dataframe.py

```diff
@@ -10,17 +10,17 @@
 
 @dataclass
 class AthenaToDataFrameSettings:
-    """S3 to Cassandra ETL settings"""
+    """S3 to Dataframe ETL settings"""
     source_database: str
     source_table: str
     source_connection_settings: AwsConnectionSettings
 
 
 class AthenaToDataFrame(S3ToDataFrame):
     """
-    Class to transfer parquet data from s3 to Cassandra
+    Class to transfer parquet data from s3 to Dataframe
     Args:
-        settings (AthenaToCassandraSettings): the settings around the etl to be executed
+        settings (S3ToDataFrame): the settings around the etl to be executed
```

Comment:
```suggestion
        settings (AthenaToDataFrame): the settings around the etl to be executed
```

Reply:
made a mistake there myself. Fixed the docs again